From f105cfcdda447f50125edf9337b7955bc30c874a Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sat, 15 Apr 2017 11:16:11 +0200
Subject: [PATCH 1/1] odroid-c2: README: MMC is supported

Mention eMMC and microSD as supported devices.

They have been enabled with patch
d0c5c8d529f16fa88ab52a3b5dd2d4fc03664f19
odroid-c2: enable new Meson GX MMC driver in board defconfig
which was accepted for u-boot-mmc.git.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
Possibly the patch should be handled via u-boot-mmc.git.
---
 board/amlogic/odroid-c2/README | 1 +
 1 file changed, 1 insertion(+)

diff --git a/board/amlogic/odroid-c2/README b/board/amlogic/odroid-c2/README
index d6d266a5ce..b407c04952 100644
--- a/board/amlogic/odroid-c2/README
+++ b/board/amlogic/odroid-c2/README
@@ -18,6 +18,7 @@ Schematics are available on the manufacturer website.
 
 Currently the u-boot port supports the following devices:
  - serial
+ - eMMC, microSD
  - Ethernet
 
 u-boot compilation
-- 
2.11.0

